transform/cities.go

```diff
@@ -9,10 +9,14 @@ import (
 	"strconv"
 )
 
-const nationalTreasureFileName = "TABMUN.CSV"
+/*
+NationalTreasureFileName is the name of the CSV containging city names and 
+their codes. 
+*/
```

Comment:
Esse não é o estilo de documentação utilizado (e tem um typo)

```suggestion
// NationalTreasureFileName is the name of the CSV containing city names and 
// their codes. 
```